src/reverb/reverb.php

```diff
@@ -789,6 +783,81 @@ protected function postProcess()
             }
         }
 
+        // Bulk sync all
+        if (Tools::isSubmit('submitBulkSyncronize_allps_product_reverb')) {
+            $fields_list = array(
+                'id_product' => array(
+                    'title' => $this->l('ID'),
+                    'width' => 30,
+                    'type' => 'int',
+                    'filter_key' => 'p!id_product'
+                ),
+                'reverb_id' => array(
+                    'title' => $this->l('Reverb ID'),
+                    'width' => 70,
+                    'type' => 'text',
+                    'filter_key' => 'rs!id_sync'
+                ),
+                'reference' => array(
+                    'title' => $this->l('SKU'),
+                    'width' => 140,
+                    'type' => 'text',
+                    'filter_key' => 'p!reference'
+                ),
+                'name' => array(
+                    'title' => $this->l('Name'),
+                    'width' => 140,
+                    'type' => 'text',
+                    'filter_key' => 'pl!name'
+                ),
+                'status' => array(
+                    'title' => $this->l('Sync Status'),
+                    'width' => 50,
+                    'type' => 'select',
+                    'search' => true,
+                    'orderby' => true,
+                    'filter_key' => 'rs!status',
+                    'badge_success' => true,
+                    'list' => array('success' => 'success',
+                        'error' => 'error',
+                        'to_sync' => 'to_sync')
+                ),
+                'details' => array(
+                    'title' => $this->l('Sync Detail'),
+                    'width' => 300,
+                    'type' => 'text',
+                    'search' => 'true',
+                    'orderby' => 'true',
+                    'filter_key' => 'details'
+                ),
+                'date' => array(
+                    'title' => $this->l('Last synced'),
+                    'width' => 140,
+                    'type' => 'datetime',
+                    'filter_key' => 'date'
+                ),
+                'reverb_slug' => array(
+                    'title' => '',
+                    'search' => false,
+                    'orderby' => false,
+                )
+            );
+            //=========================================
+            //         GET DATAS FOR LIST
+            //=========================================
+            $datas = $this->reverbSync->getListProductsWithStatus($fields_list, true, false);
```

Comment:
what does this do? Does this really try to get all of these listings from reverb via the API?

Reply:
Hi @kylecrum 
no it does not do that. This function gets in the PrestaShop database all products eligible to the listing Sync. No call Reverb api

Reply:
🥇